src/mapblock_mesh.cpp

```diff
@@ -1215,7 +1216,7 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 		// Create meshbuffer
 		// This is a "Standard MeshBuffer",
 		// it's a typedeffed CMeshBuffer<video::S3DVertex>
-		scene::SMeshBuffer *buf = new scene::SMeshBuffer();
+		scene::SMeshBufferTangents *buf = new scene::SMeshBufferTangents();
```

Comment:
Why do all mesh buffers need to be tangent mesh buffers now?  This slows down rendering regardless of the user's settings.